src/lib/features/artifacts.js

```diff
@@ -181,14 +180,14 @@ export default class Artifacts {
     if (typeof artifacts !== 'object') return;
 
     // Upload all the artifacts in parallel.
-    await Promise.all(_.map(artifacts, (value, key) => {
+    await Promise.all(artifacts.map((value, key) => {
```

Comment:
is it best to just leave this part alone? I was thinking of Object.keys(artifacts)..map(function(key, index) { .... }) but then you have to call the accessor like artifacts[key] for the value. With lodash, its much more simpler. What do you think?